charts/alfresco-search-enterprise/templates/statefulset-mediation.yaml

```diff
@@ -55,7 +54,15 @@ spec:
             httpGet:
               path: /actuator/health
               port: http
-          resources: {{- toYaml .Values.resources | nindent 12 }}
+          resources: {{- toYaml $.Values.resources | nindent 12 }}
```

Comment:
is it correct that resources are underneath the root of values?

Reply:
This is how is used to be I just enforced it after I made the `with` statement line 20 span until the end of the template